term_proposal/t0037799/proposal.md

```diff
@@ -41,3 +53,22 @@ I hope the result could be better than the plain FP32 pytorch model.
 |---|---|---|---|---|---|

```

Comment:
You should either provide a fully populated table, or explain why some cells are left empty.